### ISSUE TITLE
As a user I shouldn't be allowed to send negative credits

### ISSUE THREAD

**Issue by involve-MINT** [involveMINT/iMPublic#288]

**Comment by fogunsan** [comment]
rough code location: /involvemint/iMPublic/libs/client/shell/src/lib/wallet/wallet.component.html
To confirm: Is the code change to ensure that a negative balance doesn't show up on the UI? The current behavior is that it is not permitted to send credits when balance is 0, but @samanthap88 can also try testing when there are more than zero credits but the amount you want to send is greater than the balance.

![Image](https://github.com/involveMINT/iMPublic/assets/106701657/c43ffa1c-2b75-48c2-9bb5-7fe96ad47468)

**Comment by involve-MINT** [comment]
@sakshamg19 @fogunsan yes we want to ensure a negative balance does not show up on the UI.

In the future, we want to consider allowing negative balance in the form of a "request" but that is not part of the scope of what we are doing now.  I only say this for your future consideration when writing your code.

**Comment by mari-hirota** [comment]
Test for cases when the input is 0 (free) if there will be any error.
